src/containers/EmailConfirmationContainer.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react';
-import settings from 'settings';
+import { connect } from 'react-redux';
 import Helmet from 'react-helmet';
+import getEmailConfirmationResponse from 'selectors/emailConfirmationResponse';
 import EmailConfirmation from 'components/EmailConfirmation';
+import PetitionResponseTokenError from 'components/PetitionResponseTokenError';
 
 const EmailConfirmationContainer = React.createClass({
   render () {
+    const { displayError, translation } = this.props;
```

Comment:
I would lost the nested `translation` object and just pass `title`, `hint` etc. singularly